ui/src/components/Timeline/Timeline.test.tsx

```diff
@@ -67,6 +76,7 @@ describe('Sidebar', () => {
     await userEvent.click(swatches[3]);
     color = rows[1].style.getPropertyValue('--timeline-item-color');
     expect(color).toEqual(Colors.RED2);
+    expect(onLayoutUpdate).toHaveBeenCalledTimes(1);
```

Comment:
This expectation would fail without the fix (the callback would have been called twice).